bitwatch/umbrel-app.yml

```
@@ -0,0 +1,20 @@
+manifestVersion: 1
+id: bitwatch
+category: bitcoin
+name: Bitwatch
+version: "1.4.4"
+tagline: Monitor Bitcoin addresses in real-time
+description: >-
+  Monitor Bitcoin addresses in the mempool and on-chain using the mempool.space API. 
+  Track balance changes, get notifications, and manage expectations for multiple addresses.
+  Recommended to have Mempool + Fulcrum to run privately but not required.
```

```suggestion
description: >-
  Monitor Bitcoin addresses in the mempool and on-chain using the mempool.space API.


  Track balance changes, get notifications, and manage expectations for multiple addresses.


  Recommended to have Mempool + Fulcrum to run privately but not required.
```

Adding some space between paragraphs. Maybe the description could be extended a little further to give more context on the app.